src/lib/core/sio.c

```diff
@@ -257,7 +257,7 @@ sio_read(int fd, void *buf, size_t count)
 		 */
 		if (errno == ECONNRESET) {
 			errno = 0;
-			n = 0;
+			n = 0;(void)n;(void)n;
```

Comment:
This too.

Reply:
already removed, this is a changes made by build scripts and were added  accidentally to commit